### PR TITLE
Events landing page design

### PR DESCRIPTION
**Pull request by joypoddar**
Closes issue #105

### CODE REVIEW

File: components/common/Hero.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ import HeroLines from "../../public/common/HeroLines.svg";
 import RadialGradient from "../../public/common/RadialGradient.svg";
 import ReactPlayLogo from "../../public/ReactPlayLogo.svg";
 
-const Hero = ({ metainfo }) => {
+const Hero = ({ metainfo, theHustleHomePage }) => {
   const router = useRouter();
 
   useEffect(() => {}, [metainfo]);
```
Comment (ap221882) [inline components/common/Hero.js]: we can remove this useEffect

**Comment by joypoddar** [comment]
@atapas @ap221882 Kindly check the events, reactplay live and the twitter spaces section with reference to the latest design and let me know if I have to make it similar to the design or not.

Note: Twitter space cards and the events section have dummy elements that will be handled.

**Comment by ap221882** [comment]
I think the current design and the latest designs are different. Some minor things are also there having differences. @atapas what do you say about this?

**Comment by atapas** [comment]
> https://deploy-preview-141--activities-reactplay.netlify.app

We need to follow the design we have received from @nirmalkc 

This one:(As he also shared the spec, please follow the exact color, fonts, images specified in the spec)


image

**Comment by joypoddar** [comment]
@ap221882 Assign me to the project and you can start working on the YouTube API and adding swiperjs to the project

**Pull request merged**